Realm/Realm.Weaver/Analytics/AnalyticsUtils.cs

```diff
@@ -63,16 +64,11 @@ public static string GetTargetOsName(FrameworkName frameworkName)
                 return OperatingSystem.Linux;
             }
 
-            if (targetOs.ContainsIgnoreCase("win") || targetOs == ".NETFramework")
+            if (targetOs.ContainsIgnoreCase("win") || targetOs.ContainsIgnoreCase("net"))
```

Comment:
Wouldn't that also catch things like .NET Core, .NET standard and so on? We don't want to report those as windows anymore.

Reply:
For targetOS, we're currently reporting .NET Standard as Windows, which is misleadingly inflating the windows numbers we're seeing. Since those projects are inherently cross platform, we should instead report that to allow comparing builder numbers between Windows and the other supported OS-s.

Reply:
Ahhh, now I remember the conversation. Thank you for clarifying this.